docs/releases/1.10.txt

```diff
@@ -53,6 +53,8 @@ Minor features
   to prevent an issue where Safari caches redirects and prevents a user from
   being able to log out.
 
+* The :func:`~django.contrib.auth.views.login` view has a new parameter ``redirect_authenticated_user`` to automatically redirect users opening the view as if they had just logged in.
```

Comment:
Also, it should say "authenticated users" not just "users" -- and "opening the view" sounds like a strange usage to me, maybe "visiting the login page" instead. And I think "as if they had just logged in" is probably the default assumption anyway, and more detail than needed in release notes. So I'd just end with "...to automatically redirect authenticated users visiting the login page."